### ISSUE TITLE
A plugin is creating a recipe using a Deprecated method.

### ISSUE THREAD

**Issue by andris155** [SickSkillz/SuperLuckyBlock#7]
**Describe the bug**
A plugin is creating a recipe using a Deprecated method.

**To Reproduce**
1. Start Paper server

**Server (please complete the following information):**
 - OS: Ubuntu 16.04.1
 - Java version Java 8
 - Minecraft version: 1.13.2

**Plugin (please complete the following information):**
 - Version: 2.7.0
 - Changed config: yes
 - Added or removed custom surprises: no

**Additional notes**
```
[08:30:51] [Server thread/INFO]: [SuperLuckyBlock-Pro] Enabling SuperLuckyBlock-Pro v2.7.0
[08:30:51] [Server thread/INFO]: SLB >> Configs successfully loaded!
[08:30:51] [Server thread/INFO]: SLB >> Registering events...
[08:30:51] [Server thread/INFO]: SLB >> Registering command...
[08:30:51] [Server thread/INFO]: SLB >> Creating crafting recipe...
[08:30:52] [Server thread/INFO]: SLB >> Creating tasks...
[08:30:52] [Server thread/INFO]: SLB >> Vault found! Hooking into it...
[08:30:52] [Server thread/INFO]: SLB >> Scanning for custom surprises...
[08:30:52] [Server thread/INFO]: SLB >> Found 9 custom surprises!
[08:30:52] [Server thread/INFO]: SLB >> Initializing custom luckyblocks...
[08:30:52] [Server thread/WARN]: java.lang.Throwable: Warning: A plugin is creating a recipe using a Deprecated method. This will cause you to receive warnings stating 'Tried to load unrecognized recipe: bukkit:<ID>'. Please ask the author to give their recipe a static key using NamespacedKey.
[08:30:52] [Server thread/WARN]: 	at org.bukkit.inventory.ShapedRecipe.<init>(ShapedRecipe.java:28)
[08:30:52] [Server thread/WARN]: 	at xyz.sickskillz.superluckyblock.utils.CraftingUtils.createCraftingRecipe(CraftingUtils.java:115)
[08:30:52] [Server thread/WARN]: 	at xyz.sickskillz.superluckyblock.SuperLuckyBlock.initCustomLuckyBlocks(SuperLuckyBlock.java:132)
[08:30:52] [Server thread/WARN]: 	at xyz.sickskillz.superluckyblock.SuperLuckyBlock.onEnable(SuperLuckyBlock.java:99)
[08:30:52] [Server thread/WARN]: 	at org.bukkit.plugin.java.JavaPlugin.setEnabled(JavaPlugin.java:254)
[08:30:52] [Server thread/WARN]: 	at org.bukkit.plugin.java.JavaPluginLoader.enablePlugin(JavaPluginLoader.java:325)
[08:30:52] [Server thread/WARN]: 	at org.bukkit.plugin.SimplePluginManager.enablePlugin(SimplePluginManager.java:405)
[08:30:52] [Server thread/WARN]: 	at org.bukkit.craftbukkit.v1_13_R2.CraftServer.enablePlugin(CraftServer.java:438)
[08:30:52] [Server thread/WARN]: 	at org.bukkit.craftbukkit.v1_13_R2.CraftServer.enablePlugins(CraftServer.java:352)
[08:30:52] [Server thread/WARN]: 	at net.minecraft.server.v1_13_R2.MinecraftServer.l(MinecraftServer.java:613)
[08:30:52] [Server thread/WARN]: 	at net.minecraft.server.v1_13_R2.MinecraftServer.a(MinecraftServer.java:575)
[08:30:52] [Server thread/WARN]: 	at net.minecraft.server.v1_13_R2.MinecraftServer.a(MinecraftServer.java:435)
[08:30:52] [Server thread/WARN]: 	at net.minecraft.server.v1_13_R2.DedicatedServer.init(DedicatedServer.java:316)
[08:30:52] [Server thread/WARN]: 	at net.minecraft.server.v1_13_R2.MinecraftServer.run(MinecraftServer.java:791)
[08:30:52] [Server thread/WARN]: 	at java.lang.Thread.run(Thread.java:748)
[08:30:52] [Server thread/INFO]: SLB >> SuperLuckyBlock has been enabled! (254ms)
```

**Comment by SickSkillz** [comment]
Hello, thanks for reporting this issue.
I will look into fixing this as fast as possible!

**Comment by SickSkillz** [comment]
Fixed in version 2.7.1.
